sources/packages/backend/apps/api/src/route-controllers/program-info-request/program-info-request.institutions.controller.ts

```diff
@@ -175,8 +179,10 @@ export class ProgramInfoRequestInstitutionsController extends BaseController {
           payload.otherReasonDesc,
         );
       if (application.currentAssessment.assessmentWorkflowId) {
-        await this.workflowService.deleteApplicationAssessment(
-          application.currentAssessment.assessmentWorkflowId,
+        // Send a message to inform the workflow of the declined PIR.
+        await this.workflowClientService.sendProgramInfoCompletedMessage(
```

Comment:
I believe the sendProgramInfoCompletedMessage method in workflow-actions.service.ts can be deleted as it is not used anywhere. Or is it a plan to remove whole workflow-actions.service itself which is on API?

Reply:
`workflow-actions.service.ts` will be completely removed. I can do it as part of the next PR.